File: copy_images.js
const fs = require('fs');
const srcDir = 'C:/Users/etipa/.gemini/antigravity/brain/dc470efc-4e3a-488b-aacd-0a34c0d16fd6';
const destDir = 'C:/Users/etipa/.gemini/antigravity/scratch/kanakaiah_git/public/chapters/matthew';

for (let i = 3; i <= 18; i++) {
  const files = fs.readdirSync(srcDir);
  const match = files.find(f => f.startsWith(`matthew_ch${i}_`) && f.endsWith('.png'));
  if (match) {
    fs.copyFileSync(`${srcDir}/${match}`, `${destDir}/ch${i}.png`);
    console.log(`Copied ch${i}.png`);
  }
}

File: copy_images.py
import os
import glob
import shutil

src_dir = r"C:\Users\etipa\.gemini\antigravity\brain\c4e0d273-1f10-4c43-ad02-d1d0b66b3d91"
dest_dir = r"public\chapters\1tim"

os.makedirs(dest_dir, exist_ok=True)

files = glob.glob(os.path.join(src_dir, "1tim_ch*.png"))
for f in files:
    filename = os.path.basename(f)
    # Extract "chX" from "1tim_chX_name_id.png"
    parts = filename.split('_')
    for part in parts:
        if part.startswith("ch"):
            ch_num = part[2:]
            new_name = f"ch{ch_num}.png"
            dest_path = os.path.join(dest_dir, new_name)
            shutil.copy2(f, dest_path)
            print(f"Copied {filename} to {dest_path}")
            break
